### PR TITLE
Notebook: Save results as data set

### PR DESCRIPTION
**Pull request by SamDuan**
In this PR, I added couple lines of code in the notebook for saving result dataframe as json, csv and parquet. The code to read the result back to spark datafram is also included. 

Related story: [#46](https://github.com/kuwala-io/kuwala/issues/46)

### CODE REVIEW

**Review by mattigrthr — CHANGES_REQUESTED** [review]
We should include the three methods into the `kuwala/core/jupyter/modules/common.py`. Then we can also use it directly in the `kuwala/core/jupyter/notebooks/poi_category_by_population.ipynb` example.

File: kuwala/core/jupyter/notebooks/popularity_correlation.ipynb [hunk 1]
```diff
@@ -355,12 +363,73 @@
...
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "import re\n",
+    "for c in result.columns:\n",
+    "    result = result.withColumnRenamed(c, re.sub('[^a-zA-Z0-9\\n\\.]', '_', c))\n",
+    "result.write.format('json').save('../notebooks/output/result.json')\n",
```
[the remark this answers is not in the document]
Comment (SamDuan) [reply kuwala/core/jupyter/notebooks/popularity_correlation.ipynb:385]: Okay. Thanks for the feedback.

**Comment by SamDuan** [comment]
@mattigrthr not sure if I'm missing something. Looks like only you can trigger the workflows for the new commits. If so, could you kick them off for me?

**Comment by SamDuan** [comment]
@mattigrthr could you kick off the workflows for me again? The PR shall be ready to go.

**Review by mattigrthr — APPROVED** [review]
All tested and it's working like a charm! 👌🏽 Nicely done, @SamDuan! 🙌🏽

**Pull request merged**